source/vibe/db/postgresql/package.d

```diff
@@ -116,12 +132,26 @@ class __Conn : dpq2.Connection
             destroy(sock);
         }
 
-        auto event = createFileDescriptorEvent(sock.handle, FileDescriptorEvent.Trigger.any);
+        auto event = createFileDescriptorEvent(sock.handle, FileDescriptorEvent.Trigger.read);
```

Comment:
I do not remember exactly. It seems that when there is an write event we can also receive timeout event here.

Reply:
You still will get timeout here, I can change it back and use read only for vibe-core, so it will not break anything for sure

Reply:
I still do not understand why you talk about vibe-core...
dpq2 provides it's own `Socket` for this package also as an `Connection` too.

Please explain it for me because I am isn't familiar with vibe-core.